qa/pull-tester/rpc-tests.py

```diff
@@ -531,9 +550,56 @@ def comms(timeout):
                 except subprocess.TimeoutExpired:
                     pass
 
-                if proc.poll() is not None:
+                retval = proc.poll()
+                if not retval is None:
                     if not got_outputs[0]:
-                        comms(50)
+                        try:
+                            comms(0.1)
+                        except subprocess.TimeoutExpired as e:
+                            # print("%s: communicate timed out, but process should have been killed already!" % proc.args[0])
```

Comment:
nit: remove all commented `print`s

Reply:
I did it, but these prints both comment the code and also are useful when something goes wrong.  I guess I'm trying to say that I disagree with the idea that we shouldn't leave useful debug code in comments for use by devs who have to maintain the code later.  

In this particular case, its not documented anywhere that a core dump and only a core dump will cause this exact behavior by the python .communicate() function so I was thinking that this code may need to be re-examined in the future.

Reply:
fair enough